packages/fastui-bootstrap/src/index.tsx

```diff
@@ -58,8 +58,20 @@ export const classNameGenerator: ClassNameGenerator = ({ props, fullPath, subEle
             return 'col-md-4'
         }
       }
-    case 'FormFieldInput':
     case 'FormFieldCheckbox':
+      switch (subElement) {
```

Comment:
rather than copying logic, can we combine this with the logic below, maybe worth moving out into a separate function?

Reply:
you can probably use the `{[string]: bool}` variant of `ClassName` to make most of these cases more declarative.